mono/mini/aot-runtime.c

```diff
@@ -1932,6 +1932,18 @@ check_usable (MonoAssembly *assembly, MonoAotFileInfo *info, guint8 *blob, char
 		usable = FALSE;
 	}
 
+#ifdef MONO_ARCH_CODE_EXEC_ONLY
+	if (!(info->flags & MONO_AOT_FILE_FLAG_CODE_EXEC_ONLY)) {
+		msg = g_strdup_printf ("not compiled targeting a runtime configured as CODE_EXEC_ONLY");
```

Comment:
Minor nit, but these could be just `g_strdup()` calls.

Reply:
Thanks, this was inline with things in this function, but I can change all instances not formatting anything within their calls.